flink-core/src/main/java/org/apache/flink/core/plugin/PluginManager.java

```diff
@@ -18,42 +18,12 @@
 
 package org.apache.flink.core.plugin;
 
-import org.apache.flink.annotation.Internal;
-
-import org.apache.flink.shaded.guava18.com.google.common.collect.Iterators;
-
-import javax.annotation.concurrent.ThreadSafe;
-
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
 import java.util.Iterator;
 
 /**
- * Manager class and entry-point for the plugin mechanism in Flink.
+ * Interface for manager class and entry-point for the plugin mechanism in Flink.
```

Comment:
Maybe: PluginManager is responsible for managing cluster plugins which are loaded using separate class loaders so that their dependencies don't interfere with Flink's dependencies.

Reply:
That would be much more clear and descriptive. Thanks for the advice.